react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('Value returned by useState:', isClicked);
+  function handleClick() {
+    console.log('before setter:', isClicked);
+    setIsClicked(!isClicked);
+    console.log('After setter:', isClicked);
+  }
+  if (!isClicked) {
```

Comment:
This works, but it duplicates the code. The only difference between line 12 and line 14 is that you use 'white' in one case and `color` in the other.

See if you can figure out how to do this without all this duplicated code.